src/makeRobotAccountant.js

```diff
@@ -7,6 +7,19 @@
 
 function makeRobotAccountant() {
   // write code here
+  let callCount = 0;
+
+  return (firstValue) => {
+    callCount++;
+
+    const condition = !(callCount % 2) && callCount > 3;
+
+    return (secondValue) => (
+      condition
+        ? 'Bzzz... Error!'
+        : firstValue + secondValue
+    );
```

Comment:
```suggestion
    return (secondValue) => condition
        ? 'Bzzz... Error!'
        : firstValue + secondValue
```

Reply:
If you used ternary operator (or method) you can write it without brackets and wrapping, you can just wrap ternary operator (or method to new line)